app/scripts/views/marketing_snippet_ios.js

```diff
@@ -80,6 +85,12 @@ define(function (require, exports, module) {
       };
     },
 
+    submit: function () {
+      this.logViewEvent('sms');
+      var phoneNumber = this.getElementValue('#phone-number');
```

Comment:
We should probably do some basic validity checks on the phone number to ensure it only contains numbers, dashes, or possibly spaces and periods too if we want to be a bit more flexible, and that the number contains at most 10 digits, possibly 11 if the leading digit is a 1.